src/scenes/home/informationForm/informationForm.js

```diff
@@ -45,7 +54,18 @@ class SignupInformation extends Component {
     if (this.state.step < 1) {
       return;
     }
-    this.setState({ step: this.state.step -= 1 });
+
```

Comment:
I like the idea of clearing interests if a user navigates away from the page! In this implementation, however, if we ever add / remove a component down the line this could end up breaking because it is hard-coded via the step number. Is there a way to make this more resilient for the long term?

Reply:
```
export function areEqual(obj1, obj2) {
	return Object.keys(obj1).every((key) => obj2.hasOwnProperty(key) && (obj1[key] === obj2[key]));
};
```